### PR TITLE
53: Pull activities from https://boredapi.com

### LINKED ISSUE #53: Pull activities from https://boredapi.com
As a date-night api consumer,
When I request `GET /activities`
I want to see activities from boredapi.com included in the list
And I want all activities to use a consistent json schema

Things to consider...
- We either need to map fields from the boredapi.com activity schema to our date-night activity schema, or we could just use the schema defined by bordedapi.com activities across the board. 
- We need to determine how we'll save activities from boredapi.com. Some options:
  - We can store the boredapi.com activity id and pull the activity data when we need to display event and activity details for saved events.
  - OR we can create a date-night activity record whenever an event is saved with that activity
    - I think the most straightforward way to do this would be to have the frontend pass all activity data in the post or put to create or update events, and on the backend, we can create activity data along with the event save.
- As a first pass, we could not worry about incorporating our own date-night activities in the response data. We can come back to that when we need that functionality as there are some nuances to consider, and this work is a sizable bit of work as is.

### PR DESCRIPTION
**Pull request by msrezaie**
### Attached Issue
This PR resolves #53 

## Change Summary
- Installed Axios `npm` package. This package is primarily used for fetching data from the Bored API's URL.
- The activities data coming from BoredAPI are combined with the hard coded activities. A max limit of 6 resources is set by default.
- Accessing the `/api/v1/activities` route will return 6 randomized activity documents.

### CODE REVIEW

**Review by angelkbrown — APPROVED** [review]
This is a really nice solution, @msrezaie! Congrats on successfully integrating with a third party API. 😄 🎉 👏

File: src/controllers/activitiesController.js [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
 const getAllActivities = async (req, res) => {
-  const response = await Activity.find().populate({
-    path: "vote",
-    populate: { path: "voters" },
-  });
+  let suggestedActivities = [];
+
+  for (let i = 0; i < 3; i++) {
+    const apiResponse = await axios.get(
+      `https://www.boredapi.com/api/activity?participants=${i + 1}`
```
[the remark this answers is not in the document]
Comment (msrezaie) [reply src/controllers/activitiesController.js:12]: Yep, I had all those scenarios in mind. I even had implemented different options using query string params, though Jillian who is currently doing the frontend portion of this, didn't feel comfortable using query string params. So I had to adjust and combine all the activities coming from Bored API and our own hard coded activities into a single endpoint.

Though, I plan to continue working on our app after the MVP.

[the remark this answers is not in the document]
Comment (marice-romero) [reply src/controllers/activitiesController.js:12]: Way to think through all these options and adjusting - this is a good solution to get everything in one endpoint.

**Review by marice-romero — APPROVED** [review]
Great job, @msrezaie! In my head this was going to be super complicated but you made the solution easy to read and simple.

**Pull request merged**